crates/bevy_pbr/src/render/pbr_fragment.wgsl

```diff
@@ -170,6 +170,34 @@ fn pbr_input_from_standard_material(
         pbr_input.material.metallic = metallic;
         pbr_input.material.perceptual_roughness = perceptual_roughness;
 
+        // Clearcoat factor
+        pbr_input.material.clearcoat = pbr_bindings::material.clearcoat;
+#ifdef VERTEX_UVS
+#ifdef PBR_MULTI_LAYER_MATERIAL_TEXTURES_SUPPORTED
+        if ((pbr_bindings::material.flags & pbr_types::STANDARD_MATERIAL_FLAGS_CLEARCOAT_TEXTURE_BIT) != 0u) {
+#ifdef MESHLET_MESH_MATERIAL_PASS
+            pbr_input.material.clearcoat *= textureSampleGrad(pbr_bindings::clearcoat_texture, pbr_bindings::clearcoat_sampler, uv, in.ddx_uv, in.ddy_uv).r;
+#else   // MESHLET_MESH_MATERIAL_PASS
+            pbr_input.material.clearcoat *= textureSampleBias(pbr_bindings::clearcoat_texture, pbr_bindings::clearcoat_sampler, uv, view.mip_bias).r;
+#endif  // MESHLET_MESH_MATERIAL_PASS
```

Comment:
variants of this block are repeated 10 times now, could we factor it out?

Reply:
Done.